tests/lax_numpy_test.py

```diff
@@ -2119,6 +2119,28 @@ def testCov(self, shape, dtype, rowvar, ddof, bias, rng_factory):
     self._CompileAndCheck(lnp_fun, args_maker, check_dtypes=True, atol=tol,
                           rtol=tol)
 
+  @parameterized.named_parameters(
+      jtu.cases_from_list(
+        {"testcase_name": "_shape={}_shape2={}_dtype={}_mode={}".format(
+            shape, shape2, dtype, mode),
+         "shape": shape, "dtype": dtype, "mode": mode,
+         "shape2": shape2,"rng_factory": rng_factory}
+        for shape in [(i,) for i in range(2, 100)]
```

Comment:
This generates rather a lot of test cases (~98^2). That's too many. I think you can achieve what you want by just handwriting a few special cases, e.g.,
`[2, 3, 5, 10]`

Does it make sense to test `0` and `1` for this function? Does numpy work for those cases? If so, I'd test them too.